src/ipc/ipc4/handler.c

```diff
@@ -727,6 +727,7 @@ static int ipc4_set_large_config_module_instance(struct ipc4_message_request *ip
 	int ret;
 
 	memcpy_s(&config, sizeof(config), ipc4, sizeof(config));
+	dcache_invalidate_region((__sparse_force void __sparse_cache *)MAILBOX_HOSTBOX_BASE, config.extension.r.data_off_size);
```

Comment:
sorry, don't understand. `MAILBOX_HOSTBOX_BASE` is where the response is written, right? `ipc4_set_large_config_module_instance()` is just a normal IPC processing function, why does it need to invalidate cache on that area and other IPC processing functions don't need to do that?

Reply:
Its not response, there is data for component set_large_config
ret = drv->ops.set_large_config(..., (char *)MAILBOX_HOSTBOX_BASE);

and this is not unique case, in the same file all cases with MAILBOX_HOSTBOX_BASE will invalidate too